frontend/src/templates/Field/CountryRegionField/CountryRegionField.tsx

```diff
@@ -0,0 +1,40 @@
+import { CountryRegion } from '~shared/constants/countryRegion'
+import {
+  BasicField,
+  CountryRegionFieldBase,
+  FormFieldWithId,
+} from '~shared/types/field'
+
+import { createCountryRegionValidationRules } from '~utils/fieldValidation'
+import DropdownField from '~templates/Field/Dropdown'
+
+import { BaseFieldProps } from '../FieldContainer'
+
+export type CountryRegionFieldSchema = FormFieldWithId<CountryRegionFieldBase>
+export interface CountryRegionFieldProps extends BaseFieldProps {
+  schema: CountryRegionFieldSchema
+}
+
+// Exported for testing
```

Comment:
`nit:` Perhaps this should be moved to another file. Some sort of utility file. Or put with the enum?

No action will be taken for now since there is no clear decision on what to do with this.

Reply:
no need ba, this is a frontend-only construct so don't need to put with enum. Still ok to put here for now imho